src/lib.rs

```diff
@@ -193,6 +193,19 @@ use proc_macro2::Span;
 #[doc(hidden)]
 pub mod macros;
 
+/// Changes how bounds are addeda
+#[derive(Debug, Copy, Clone, PartialEq, Eq, Hash)]
+pub enum AddBounds {
+    /// Add for fields and generics
+    Both,
+    /// Fields only
+    Fields,
+    /// Generics only
+    Generics,
+    /// None
+    None,
```

Comment:
I think I'd like to include a nonexhaustive variant in case we choose to add more bounds options in the future :-)

```suggestion
    None,
    #[doc(hidden)]
    __Nonexhaustive,
```